sockeye/encoder.py

```diff
@@ -761,6 +782,40 @@ def append(self, cls, infer_hidden: bool = False, **kwargs) -> Encoder:
         return encoder
 
 
+class EmptyEncoder(Encoder):
+    """
+    This encoder doing nothing.
```

Comment:
Could be: "This encoder ignores the input data and simply returns zero-filled states in the expected shape."